### PR TITLE
Fix broken osd functional test repo

### PR DESCRIPTION
**Pull request by riysaxen-amzn**
### Description
This PR will fix the broken osd repo functional tests

### Issues Resolved
Testing

https://github.com/opensearch-project/dashboards-notifications/assets/69919272/9538ea20-f5d4-4b56-ba58-c0db486ad4b7

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: public/pages/Main/Main.tsx [hunk 1]
```diff
@@ -78,7 +79,7 @@ export default class Main extends Component<MainProps, MainState> {
           'ses_account',
           'email_group',
         ],
-        tooltipSupport: serverFeatures.tooltipSupport,
+        tooltipSupport: false,
```
Comment (yujin-emma) [inline public/pages/Main/Main.tsx:82]: Is this a tmp fix?

Comment (riysaxen-amzn) [reply public/pages/Main/Main.tsx:82]: not really, this is when serverFeatures is null, we should assign some boolean to tooltip

**Comment by yujin-emma** [comment]
Could you please add a video in the description to show the notification is healthy with the fix

**Comment by BionIT** [comment]
Can we fix the reference in server folder, eg. https://github.com/opensearch-project/dashboards-notifications/pull/188/files#diff-6495f662f11eb9366b891c845ddac043c64b34f3a11695a2bdb0d55f58169d4cR14

**Review by BionIT — APPROVED** [review]
LGTM, we can wait until https://github.com/opensearch-project/dashboards-notifications/actions/runs/8824156128/job/24226041378?pr=189 succeed before merging

**Review by yujin-emma — APPROVED** [review]
Overall look good to me, since the `CHANNEL_TYPE` and `BACKEND_CHANNEL_TYPE` are the only constant that used in server folder but defined in public folder previously, this fix will be fine for now. But for long term, I prefer to moving all the shared utils under common folder.

**Comment by riysaxen-amzn** [comment]
> Overall look good to me, since the `CHANNEL_TYPE` and `BACKEND_CHANNEL_TYPE` are the only constant that used in server folder but defined in public folder previously, this fix will be fine for now. But for long term, I prefer to moving all the shared utils under common folder.

i agree

**Review by AWSHurneyt — APPROVED** [review]

**Review by sbcd90 — APPROVED** [review]

**Comment by SuZhou-Joe** [comment]
Should close #185

**Comment by BionIT** [comment]
Could we backport this fix to 2.x? @AWSHurneyt @SuZhou-Joe

**Comment by SuZhou-Joe** [comment]
@BionIT Should backport this PR after #181. And @amsiglan @riysaxen-amzn could you please help to backport as you have more context than us and we are afraid we may make an unnecessary backport by accident.

**Pull request merged**